Gemfile.lock

```diff
@@ -47,7 +47,7 @@ GEM
       multi_json (~> 1.0)
     addressable (2.3.5)
     arel (3.0.3)
-    bcrypt (3.1.7)
+    bcrypt (3.1.10)
```

Comment:
All of the above is what I see too. Yeah I don't know why installing a new gem causes bundler to update existing ones - it could just keep everything else the same unless it _needs_ to update one because the new gem depends on a different version. But ok, for whatever reason, the behaviour is "install the new gem and update all of its dependencies".

So given that, the remaining question is: why does it update bcrypt, given it's not in the dependency chain of `will_paginate_mongoid`? Well `bcypt` _is_ a dependency of `ActiveModel`, but it's deliberately not in `ActiveModel`'s gemspec (see [here](https://github.com/rails/rails/blob/4-2-stable/activemodel/lib/active_model/secure_password.rb#L58)). So how would Bundler know about it?? I don't know.

Anyway in the end I refer back to my original comment: I don't think we should keep the updates to the other gems. We should add `will_paginate_mongoid` with the minimum changes necessary. So I would follow option 3 in the makandra link above - munge `Gemfile.lock` so that the only change is the addition of `will_paginate_mongoid`.